lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -127,15 +127,19 @@ defmodule Mix.Compilers.Elixir do
 
     {compile, runtime} = Kernel.LexicalTracker.remotes(module)
```

Comment:
thanks, I will fix it. Just to double-check, a "remote" is always just a module atom, right? Keeping LexicalTracker.remotes existing API would make these mix changes unnecessary which would be good too.